### ISSUE TITLE
Godot does not use the preferred GPU selected in Windows 10 Graphics Settings

### ISSUE THREAD

**Issue by Error7Studios** [godotengine/godot#47966]
**Godot version:**
3.2.3 - 3.3 RC9

**OS/device including version:**
Windows 10, GLES 2 & GLES 3

```
Graphics
		Monitor
			Name	BenQ GL2760 on NVIDIA GeForce GTX 1050 Ti
			Current Resolution	1920x1080 pixels
			Work Resolution	1920x1080 pixels
			State	Enabled, Primary
			Monitor Width	1920
			Monitor Height	1080
			Monitor BPP	32 bits per pixel
			Monitor Frequency	60 Hz
			Device	\\.\DISPLAY1\Monitor0
		Intel HD Graphics 630
			Manufacturer	Intel
			Model	HD Graphics 630
			Device ID	8086-5912
			Revision	5
			Subvendor	Lenovo (17AA)
			Current Performance Level	Level 0
			Current GPU Clock	0 MHz
			Voltage	0.675 V
			Driver version	26.20.100.7986
				Count of performance levels : 1
					Level 1 - "Perf Level 0"
		NVIDIA GeForce GTX 1050 Ti
			Manufacturer	NVIDIA
			Model	GeForce GTX 1050 Ti
			Device ID	10DE-1C82
			Revision	A2
			Subvendor	EVGA (3842)
			Current Performance Level	Level 0
			Current GPU Clock	961 MHz
			Current Memory Clock	3504 MHz
			Current Shader Clock	3504 MHz
			Voltage	0.675 V
			Technology	14 nm
			Bus Interface	PCI Express x16
			Temperature	39 °C
			Driver version	27.21.14.6140
			BIOS Version	86.07.39.00.52
			Memory	4095 MB
				Count of performance levels : 1
						Level 1 - "Perf Level 0"
							GPU Clock	961 MHz
							Shader Clock	3504 MHz
```

**Issue description:**
Godot does not use the preferred GPU selected in Windows 10 Graphics Settings.

**Steps to reproduce:**
I have selected Power Saving mode in Windows settings for Godot, which should use my integrated Intel graphics card.

![Graphics Preference](https://user-images.githubusercontent.com/60025044/115097448-eae23e00-9eef-11eb-8160-073f3c94288c.png)

When running Godot, it tells me that it is using my NVIDIA graphics card.
```
--- Debugging process started ---
Godot Engine v3.3.rc9.official - https://godotengine.org
OpenGL ES 3.0 Renderer: GeForce GTX 1050 Ti/PCIe/SSE2
OpenGL ES Batching: ON
```

Settings Windows 10 to Power Saver mode makes no difference.

![Power Saver Mode Enabled](https://user-images.githubusercontent.com/60025044/115097485-24b34480-9ef0-11eb-810a-3082c87323a2.png)

**Minimal reproduction project:**
N/A

**Comment by Calinou** [comment]
@Error7Studios Are you using a desktop or laptop computer?

**Comment by Error7Studios** [comment]
I'm using a desktop.
The graphics setting in UEFI was set to Auto (PEG first, then IGP).
Disabling the NVIDIA driver didn't work. I would just get this error when I tried to run Godot.

![Cant Initialize Video Driver](https://user-images.githubusercontent.com/60025044/115166204-8b169f00-a077-11eb-8a06-80fdad413315.png)

Changing the UEFI setting to IGP did the trick.
Thanks for the help. Closing.
